src/linker/Linker.Dataflow/Helpers.cs

```diff
@@ -0,0 +1,23 @@
+using System.Runtime.CompilerServices;
+
+using Mono.Cecil;
+
+namespace Mono.Linker.Dataflow
+{
+	static class TypeHelper
```

Comment:
I don't think we need this type. If you really need the one lines add them to the existing extension types

Reply:
This is also in connection with "ValueNode.cs has barely any references to Cecil - the hope is that we would be able to reuse this file for e.g. a Roslyn analyzer" - if we decide we don't care, I can inline this into ValueNode. It's only used from ValueNode.